lib/Cache/RedisDB.pm

```diff
@@ -133,6 +133,34 @@ sub del {
     return redis->del(map { "${namespace}::$_" } @keys);
 }
 
+=head2 keys($namespace)
+
+Return a list of all known keys in the provided I<$namespace>.
+
+=cut
+
+sub keys {
+    my ($self, $namespace) = @_;
+    my $prefix = $namespace . '::';
```

Comment:
Given that this means undef will throw a warning, would it be worth documenting this as being required and croaking if $namespace is undef?  Just a thought.